ApplicationLibCode/ReservoirDataModel/RigGridBase.cpp

```diff
@@ -218,6 +218,34 @@ bool RigGridBase::ijkFromCellIndex( size_t cellIndex, size_t* i, size_t* j, size
     return true;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::optional<std::array<size_t, 3>> RigGridBase::ijkFromCellIndex( size_t cellIndex ) const
+{
```

Comment:
Could also make a IJK class? And have it handle the change from zero to one-indexed?

Reply:
`caf::VecIjk `already exists, and could be used?

Reply:
Yes, now using caf::VecIjk